source/common/protobuf/utility.cc

```diff
@@ -10,6 +10,7 @@
 #include "source/common/common/assert.h"
 #include "source/common/common/documentation_url.h"
 #include "source/common/common/fmt.h"
+#include "source/common/json/json_streamer.h"
```

Comment:
can we move the new utility that depends on JsonStreamer into the json library? I think it's confusing, at the directory level, to have json depend on proto and vice versa. I don't think it's hard to break the cycle by moving the new utility.

Reply:
SGTM.